scripts/repos/cray.template.repos

```diff
@@ -7,10 +7,17 @@ https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifac
 https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sat-rpms/hpe/stable/sle-15sp3?auth=basic                                                                                      cray-algol60-sat-rpms-stable-sle-15sp3                                                     --no-gpgcheck -p 88
 
 # COS
-https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/dst-rpm-mirror/cos-rpm-stable-local/release/cos-2.5/sle${releasever_major}_sp${releasever_minor}_ncn?auth=basic               cray-cos-sle-2.5-SHASTA-OS-cos-ncn                                                         --no-gpgcheck -p 89
+#https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/dst-rpm-mirror/cos-rpm-stable-local/release/cos-2.5/sle${releasever_major}_sp${releasever_minor}_ncn?auth=basic              cray-cos-sle-2.5-SHASTA-OS-cos-ncn                                                         --no-gpgcheck -p 89
+
+#TEMP
+https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/dst-rpm-mirror/cos-rpm-stable-local/release/cos-2.5/sle15_sp4_ncn?auth=basic                                                  cray-cos-sle-2.5-SHASTA-OS-cos-ncn                                                         --no-gpgcheck -p 89
```

Comment:
Does COS not have an SP5 repo yet, or is it just missing a select few packages? (Asking without checking, not to be annoying).

I'd recommend just _adding_ instead of _replacing_ the repo, and set it to a lower priority (90). Then Zypper can choose to pull from the SP5 repo as COS publishes packages, and what it can't resolve it'll resolve from the legacy SP4 repo.

Reply:
COS doesn't have a stable 2.6 repo (SP5) yet and they don't have a dev or stable repo for SP5 NCN's. It felt like the best move was to stick with SP4.